packages/react/src/components/Tree/Tree.tsx

```diff
@@ -151,6 +158,18 @@ class Tree extends AutoControlledComponent<WithAsProp<TreeProps>, TreeState> {
         {childrenExist(children) ? children : this.renderContent()}
       </ElementType>
     )
+
+    return contentRef ? (
+      <Ref
```

Comment:
what if you wrap the Tree comp with a <Ref> in TreeItem? So then you render <Ref> only for subtree and no need to have this check here and pass a prop?